operators/multiclusterobservability/manifests/base/alertmanager/alertmanager-operated.yaml

```diff
@@ -2,14 +2,10 @@ apiVersion: v1
 kind: Service
 metadata:
   name: alertmanager-operated
-  namespace: open-cluster-management
+  namespace: open-cluster-management-observability 
```

Comment:
Is this change necessary? Is it breaking? 🤔

Reply:
All those namespaces are reset by the controller. So personally, I would remove all of them from the manifests. But I am not sure it has not impact 🤷 
Resetting this one as it wasn't changed on purpose

Reply:
I removed all unnecessary namespaces. Unit tests are checking them now